client/src/components/RatingsAndReviews/RatingsAndReviews.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import './RatingsAndReviews.scss';
 import Sorting from './Sorting/Sorting.jsx';
 import ReviewsList from './ReviewsList/ReviewsList.jsx';
-import WritingReview from './WritingReview/WritingReview.jsx';
+// import WritingReview from './WritingReview/WritingReview.jsx';
```

Comment:
If you don't need it, removing this will keep the code base clean in the main branch if you are merging the branch.

Reply:
done.